apps/server/src/stores/runtimeState.ts

```diff
@@ -93,7 +95,8 @@ export function clear() {
   runtimeState.publicEventNext = null;
 
   runtimeState.runtime.offset = 0;
-  runtimeState.runtime.actualStart = null;
+  runtimeState.runtime.relativeOffset = 0;
```

Comment:
question: had we agreed to create two variants of the clear function like
softClear() and hardClear()?